### DISCUSSION TITLE
Can you remove the flare skill gauge that goes off when you finish a set of songs becaus e it causes the game to crash. DDR WORLD Aspherxia DDR Plugin

### DISCUSSION THREAD

**Discussion by makoflagkk** [community/community#128962] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Remove it

**Comment by ebndev** [comment]
Hi @makoflagkk, thanks for being a part of the GitHub Community!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from specific project repo, is that right?

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

If you don't mind, would you mind letting us know how you ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your Discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board. Any insight would be helpful, thanks!

**Reply by makoflagkk** [reply]
I thought there was a way you could comment under someone's repo profile even if it was closed sourced. But I wanted to report an issue but the repo doesn't have a issues tab.

**Reply by ebndev** [reply]
Thanks for getting back to me!

The “Send feedback” on a user’s profile is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions. You might want to try to contact them on their website or social media if they have any? 

Unfortunately there is currently no way to contact users directly on GitHub and this is a public forum.
